Lesson21/todo-app/sahra/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>To-Do App</title>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+  </head>
+  <body>
+    <main class="container">
+      <form class="todo-form" id="todoForm">
+        <h1><label for="todoInput">To-Do List</label></h1>
+        <input placeholder="Add to do item" id="todoInput" class="todo-form-item"/>
```

Comment:
Placeholder should show example of expected input, e.g. 'Do homework' would be a better placeholder value in this case